src/games/gameitem.py

```diff
@@ -268,13 +341,17 @@ def _player_table_row(self, player, align):
 
     def _country_icon_fmt(self, player):
         icon_path_fmt = os.path.join("chat", "countries", "{}.png")
-        icon_path = icon_path_fmt.format(player.country.lower())
+        if player.country is None or player.country == '':
+            country = '__'
+        else:
+            country = player.country.lower()
+        icon_path = icon_path_fmt.format(country)
         icon_abs_path = os.path.join(util.COMMON_DIR, icon_path)
         return "<img src='{}'>".format(icon_abs_path)
 
     def _player_fmt(self, player):
         if player == self._me.player:
-            pformat = "<b><i>{}</b></i>"
+            pformat = "<b><i>{}</i></b>"
```

Comment:
This seems like the sort of obvious fix that could be its own commit, and merged independently of this giant monolithic commit.

Giant monolithic commits are extremely hard to review. This may be why so many of the long-open PRs on this repository are the ones that are like that :D